components/servo/lib.rs

```diff
@@ -237,10 +244,46 @@ impl<Window> Browser<Window> where Window: WindowMethods + 'static {
         Browser {
             compositor: compositor,
             constellation_chan: constellation_chan,
+            embedder_receiver: embedder_receiver,
+        }
+    }
+
+    fn handle_window_event(&mut self, event: WindowEvent) {
+        match event {
+            WindowEvent::Reload => {
+                let top_level_browsing_context_id = match self.compositor.root_pipeline {
```

Comment:
Eventually, the embedder will have a list of `top_level_browsing_context_id` (tabs).